shared/src/main/scala/com/cibo/evilplot/colors/Coloring.scala

```diff
@@ -47,7 +47,11 @@ trait CategoricalColoring[A] extends Coloring[A] {
     distinctElemsAndColorFunction(dataToColor)._2
   }
 
-  def legendContext(dataToColor: Seq[A])(implicit theme: Theme): LegendContext = {
+  def legendContext(dataToColor: Seq[A])(implicit theme: Theme): LegendContext =
+    legendContext(dataToColor, legendGlyph = (d: Double) => Disc(d))
+
+  def legendContext(dataToColor: Seq[A], legendGlyph: Double => Drawable)(
+    implicit theme: Theme): LegendContext = {
     val (distinct, coloring) = distinctElemsAndColorFunction(dataToColor)
     LegendContext(
       elements = distinct.map(v => Disc(theme.elements.pointSize) filled coloring(v)),
```

Comment:
Should this be `legendGlyph(theme.elements.pointSize)' instead of using Disc everytime?